gwpy/frequencyseries/frequencyseries.py

```diff
@@ -435,6 +435,10 @@ def to_pycbc(self, copy=True):
             epoch = None
         else:
             epoch = self.epoch.gps
+        if self.f0.to('Hz').value > 0.0:
```

Comment:
Since the requirement is that the `f0` be identically zero, we should just check the truthiness of the value:

```suggestion
        if self.f0.to('Hz').value:
```